feature/notification/src/main/java/io/github/droidkaigi/confsched2020/notification/service/FCMService.kt

```diff
@@ -0,0 +1,92 @@
+package io.github.droidkaigi.confsched2020.notification.service
+
+import android.app.PendingIntent
+import android.content.Intent
+import android.os.Build
+import androidx.core.os.bundleOf
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import io.github.droidkaigi.confsched2020.notification.NotificationChannelInfo
+import io.github.droidkaigi.confsched2020.notification.NotificationUtil
+import io.github.droidkaigi.confsched2020.notification.createDefaultNotificationChannel
+import io.github.droidkaigi.confsched2020.timber.error
+import timber.log.Timber
+
+class FCMService : FirebaseMessagingService() {
```

Comment:
Thanks!
I think this is not a related code.
Class for receiving announcements from DroidKaigi.